tests/tracking/test_client.py

```diff
@@ -75,28 +70,59 @@ def test_client_create_run_overrides(mock_store):
     )
 
 
-def test_client_search_runs(mock_store, mock_search_filter):
-    experiment_ids = [mock.Mock() for _ in range(5)]
+class SearchFilterMatcher:
```

Comment:
I switched the mock of SearchFilter to a matcher pattern in order to assert on the exact value of the filter.